README.md

```diff
@@ -52,25 +27,25 @@ satisfied.
 
 Otherwise, please run the following command line to install the collection.
 
-```
+```bash
```

Comment:
@richm this Collection requirements section comes from template's README, right? We do not copy it automatically. It's identical in many roles so better to ask.

Reply:
> @richm this Collection requirements section comes from template's README, right?

Right.

> We do not copy it automatically.

Right.

> It's identical in many roles so better to ask.

If it is identical, that means it didn't need to be changed.